src/promptflow/promptflow/_sdk/operations/_flow_operations.py

```diff
@@ -236,7 +236,8 @@ def _dump_connection(self, connection, output_path: Path):
         else:
             secret_dict = connection_yaml
 
-        env_var_names = [f"{connection.name}_{secret_key}".upper() for secret_key in connection.secrets]
+        connection_var_name = connection.name.replace(" ", "")
```

Comment:
not sure if we should extract a function for this given it's a magic operation and referred in 2 places.
And maybe replace it with "_" instead of " ".